docs/source/technical_tutorials/visualization/visualize_lonboard.ipynb

```diff
@@ -0,0 +1,574 @@
+{
```

Comment:
<p>stickler point, but some uniform spacing here. Missing space after <code>Shinde</code> and before <code>Sheyenne</code>.</p><p> </p><p>We should also explain that lonboard allows to visualizing these field measurement at scale.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>